evm_arithmetization/src/all_stark.rs

```diff
@@ -120,6 +132,88 @@ impl Table {
             Self::MemAfter,
         ]
     }
+
+    /// Returns all STARK table indices in descending order of their padded
+    /// trace degrees.
+    pub(crate) const fn all_sorted() -> [Self; NUM_TABLES] {
```

Comment:
Nit: this is called a lot of times. May be better to initialize it and then reuse the var instead of recomputing.